devsetup/scripts/edpm-compute-repos.sh

```diff
@@ -30,14 +30,16 @@ if [[ ! -f $SSH_KEY ]]; then
 fi
 
 cat <<EOF > $CMDS_FILE
-rpm -q git || sudo yum -y install git
-sudo yum -y install python-setuptools python-requests python3-pip
-git clone https://github.com/openstack-k8s-operators/repo-setup
-pushd repo-setup
-sudo pip install -r requirements.txt
-sudo python3 setup.py install
+# Install latest version of repo-setup without installing pip or git
+pushd /tmp
+curl -sL https://github.com/openstack-k8s-operators/repo-setup/archive/refs/heads/main.tar.gz | tar xvz
```

Comment:
My security hat is tingling. Would there be a way to get some checksum or anything just to ensure the archive is valid before un-taring it?

Reply:
If we're going to go that route, is there a way we can easily get the checksum in order to compare? We shouldn't hard code it in the script as it will change very often.

Reply:
I think there is a known URI, but I can't figure it out yet. But yeah, we won't hard-code the checksum, that's certain.

Reply:
I can't see how to fetch a checksum for an archive from github. I'm not sure it would give any MITM protection anyway since it would also be from github.com. If repo-setup releases were tagged we could switch to tag downloads and hard coded checksums. Given this is for dev setups I'm not sure its worth too much effort.

What I'd *really* like is the latest repo-setup packaged as a simple rpm, and published to a short stable URL

Reply:
I've seen many mention of such checksum, but no URI :/. weird.
It was more in the case of some network issue where the tar gets corrupted - indeed if some rogue MitM happens, over TLS, fetching the checksum from the same location would be meaningless.

I don't think there's any plan on providing RPM for that package, but we may reconsider this in the future - maybe something to bring to the team.

Resolving this conversation.